src/components/Posts/ComposeReply.tsx

```diff
@@ -37,6 +38,7 @@ const styles = {
     padddingY: 0,
     justifyContent: "space-between",
   },
+  emojiContainer: { position: "absolute", zIndex: 1, marginTop: 5 },
 };
```

Comment:
we should adjust the style of the picker:


```suggestion
  emojiPicker: {
    height: 300,
    width: 250,
    "--epr-emoji-size": "20px",
  },
```

Reply:
I will adjust the size a bit more

Reply:
Adjusted to 25.